src/modal/Modal.svelte

```diff
@@ -1,142 +1,177 @@
 <script lang="ts">
-  import { onMount } from "svelte"
-  import type { StarknetWindowObject } from "@starknet-io/types-js"
-  import ConnectorButton from "./ConnectorButton.svelte"
-  import type { StarknetkitConnector } from "../connectors/connector"
-  import { InjectedConnector } from "../connectors/injected"
+  // import { onMount } from "svelte"
```

Comment:
all these comments are placeholders or can be removed?

Reply:
wip